s4e-web/src/app/utils/s4e-date/s4e-date.pipe.ts

```diff
@@ -12,6 +12,9 @@ export class S4EDatePipe extends DatePipe {
   }
 
   transform(value: any, args?: any): any {
+    if (value == null) {
+      return '';
+    }
     return super.transform(moment.utc(value, environment.backendDateFormat).toDate());
```

Comment:
Wouldn't it be better to write:

```
return value == null ? '' : super.transform(moment.utc(value, environment.backendDateFormat).toDate());

```

Are you sure that a double comparison sign is enough? Why aren't you using triple one?

Reply:
Yes, here we explicitly want to catch `null` and `undefined`